### PR TITLE
process Variables and Plan pages

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,9 +5,26 @@ This is tests which meant to cover https://github.com/SUSE-Enceladus/blue-horizo
...
+tenant_id=...
+ssh_username=...
+ssh_public_key=...
+cluster_admin_password=...
+uaa_admin_client_secret=...
+dns_zone_name=...
+cap_domain=...
+email=...
+[cluster_labels]
+
```
Comment (asmorodskyi) [inline README.md:27]: add 'terraform apply'

File: test_aks.py [hunk 2]
```diff
@@ -3,18 +3,44 @@
...
     config.read('test.properties')
+    keys = ["subscription_id", "resource_group", "location", "client_id", "client_secret", "tenant_id",
+            "ssh_username", "ssh_public_key", "cluster_admin_password", "uaa_admin_client_secret", "dns_zone_name", "cap_domain", "email"]
+    return dict(map(lambda x: (x, config.get('variables', x)), keys))
+
+
+@pytest.fixture
+def login():
+    config = configparser.RawConfigParser()
+    config.read('test.properties')
```
Comment (cfconrad) [inline test_aks.py:22]: The README mentions `.properties` but we are reading from `test.properties`

File: README.md [hunk 3]
```diff
@@ -5,9 +5,31 @@ This is tests which meant to cover https://github.com/SUSE-Enceladus/blue-horizo
...
+
+```
+2. Create System Under Test with :
+```
+# cd  ./terraform
```
Comment (cfconrad) [inline README.md:32]: `terraform init` first

File: test_aks.py (hunk 2 above)
Comment (asmorodskyi) [reply test_aks.py:23]: done

**Pull request merged**